acceptance-tests/src/main/java/io/blueocean/ath/pages/classic/LoginPage.java

```diff
@@ -35,36 +35,24 @@ public LoginPage(WebDriver driver) {
     }
 
     public static String getUsername() {
-        return "alice";
+        return System.getProperty("adminUsername", "alice");
     }
 
     public static String getPassword() {
-        return "alice";
+        return System.getProperty("adminPassword", "alice");
     }
 
-    @Inject
-    WaitUtil wait;
     public void open() {
-        driver.get(base + "/login");
+        go(base + "/login");
         Assert.assertEquals(base + "/login", driver.getCurrentUrl());
     }
 
     public void login() {
         open();
-
-
-        WebElement usernameField = wait.until(By.id("j_username"));
-        usernameField.sendKeys(getUsername());
-
-        wait.until(By.name("j_password")).sendKeys(getPassword());
-
-        wait.until(By.xpath("//*/button[contains(text(), 'log')]")).click();
-        wait.until(driver -> {
-            if(driver.getCurrentUrl().contains("loginError")) {
-                throw new RuntimeException("Error logging in");
-            }
-            return ExpectedConditions.urlToBe(base + "/").apply(driver);
-        });
-        logger.info("Logged in as alice");
+        find("input[name=j_username]").sendKeys(getUsername());
+        find("input[name=j_password]").sendKeys(getPassword());
+        find("//button[contains(text(), 'log')]").click();
```

Comment:
I like this shorthand overall. Per discussion in Gitter, I think we want a static `click` method that implements the same retry logic as here:  https://github.com/jenkinsci/blueocean-plugin/pull/1412/files#diff-9d7946f8ccd8a4191146940351b51999

Reply:
Yeah, absolutely. This is why I said it was a 30 minute implementation. The real logic typically ends up being more complex, and yeah, definitely agree what we do is add utility methods like `click` with retry and `setText` which should be doing what `sendKeys` is doing now, e.g. you just want to set the text of an input, but you may want to send individual keys for other purposes e.g. testing an autocomplete